CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ Mapbox welcomes participation and contributions from everyone.
 
 ## Dependencies
 * Update gl-native to v10.3.0, common to v21.1.0. ([#1105](https://github.com/mapbox/mapbox-maps-android/pull/1105))
+* Introduce experimental apis for integrating android auto. `MapboxCarMap` and `MapboxCarMapObserver`. ([#1004](https://github.com/mapbox/mapbox-maps-android/pull/1004))
```

Comment:
Do we need to have auto extension entries in the changelog once it hasn't yet been release publicly?
wdyt @tobrun

Reply:
Leads to another question with releases. Can we have this release with the next train? I just uncommented the publisher. If we decide to not publish, will also remove the changelog 
```
from("$rootDir/gradle/sdk-registry.gradle")
```